js/interface.js

```diff
@@ -567,12 +655,26 @@ function saveCurrentData() {
     return Fliplet.DataSources.update(currentDataSourceId, { definition: dataSource.definition });
   }).catch(console.error);
 
-  currentDataSourceUpdatedAt = moment().format('MMM Do YYYY, HH:mm');
+  currentDataSourceUpdatedAt = TD(new Date(), { format: 'lll', locale: locale });
+
+  var replace = false; // Set to TRUE to commit the changes by replacing all existing entries
```

Comment:
@squallstar Yes, potentially. I'm currently waiting for your assessment on the commit endpoint to determine what's the best way of using this. For example, if all the rows are to be updated, it could be faster to set `replace = true` to use a different payload. It hasn't been decided yet.